tests/test_parser.py

```diff
@@ -168,6 +168,8 @@ def test_string_literal_concat(first_prefix, second_prefix, check_ast):
 
 
 def test_f_env_var(check_xonsh_ast):
+    if VER_MAJOR_MINOR > (3, 11):
+        pytest.xfail("f-string with special syntax are not supported yet")
```

Comment:
Can we open a tracking issue for this so it doesn't get missed?

Reply:
https://github.com/xonsh/xonsh/issues/5166 done